crypto/ocsp/ocsp_test.cc

```diff
@@ -1088,3 +1599,116 @@ TEST(OCSPTest, TestNotValidResponseOCSP) {
                             &basic_response,
                             &server_cert_chain);
 }
+
+
+// Test valid OCSP response hashed with sha256
+TEST(OCSPTest, TestGoodOCSP_SHA256) {
```

Comment:
Nit: the SHA1 and SHA256 OCSPTest code are relatively duplicate. Do we consider to refactor the code?

Reply:
Yes, I think we should. I already have a PR ready to refactor the `ocsp_test.cc` test file after all the tests are imported :)